lib/ret.ex

```diff
@@ -1,9 +1,207 @@
 defmodule Ret do
   @moduledoc """
-  Ret keeps the contexts that define your domain
-  and business logic.
+  The boundary of the Ret context.
+  """
+  alias Ret.{Account, Asset, Avatar, AvatarListing, OwnedFile, Project, Repo, Scene, SceneListing, Storage}
+  import Canada, only: [can?: 2]
+  import Ret.Schema, only: [is_serial_id: 1]
+  require Ecto.Query
+
+  @doc """
+  Obliterates the account with the given `account_id` and all its owned assets.
+
+  Returns `:ok` if successful, otherwise it returns `{:error, reason}`.
+
+  Error reasons are:
+
+  * `:forbidden` - the current user does not have sufficient authorization or
```

Comment:
While I do think its right that an unauthenticated user should not be able to find out if an account is valid or not, an authenticated user should be able to differentiate between an auth error and a non existing account.